### DISCUSSION TITLE
Expose country flag and code service

### DISCUSSION THREAD

**Discussion by Khousheish** [primefaces/community#1105] [PrimeNG] [answered]
Hi,
In my app i want to show a list of countries with their flags and codes.
I noticed that within the primeng showcase there is a nice service(CountryService) which has a list of countries and their respective ISO codes already.
Is there a way to expose this service so i can use it in my app rather than copy pasting the list in my app and worrying that it might get outdated at any point?
I understand that that service is mainly for mocking purposes, however it does seem to be a nice addition to the api module primeng exposes😁.
I am open to opening a PR to expose these endpoints, but i would like to see first what others think about this and if there is anyone else who has this usecase and how you might have solved it for your app.

**Comment by Chrishalaby** [comment]
I have had this issue too and agree

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the PR, could you please share the PR link here so we can take a look at it?

**Reply by Khousheish** [reply]
Hi @cetincakiroglu , I have not opened a PR yet. I wanted to get your thoughts first to see if this is something that should be added or not. If I have the time, I will try to open a PR today and will share it here once it's done.

**Comment by Khousheish** [comment]
@cetincakiroglu Initial PR created here!
https://github.com/primefaces/primeng/pull/14717

**Comment by Khousheish** [comment]
Hi @cetincakiroglu any timeline on when the PR can get a quick look? I need this for my project. If it will take some time let me know so i take action please!

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

It's a dummy service for demo purposes we do not need to make it public. Doesn't affect anything in the library. If you need in your project, you can create your own service.
